pkg/volume/util/device_util_linux.go

```diff
@@ -0,0 +1,86 @@
+// +build linux
```

Comment:
Is there anything here that wouldn't work for `darwin` (i.e. Mac)? Also create an "unsupported" version to gracefully handle unsupported builds (see `pkg/volume/util/fs_unsupported.go`, for example).